src/functionalTest/resources/features/F-004/F-004.feature

```diff
@@ -0,0 +1,118 @@
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-046
+  Scenario: must get an error response for a malformed classification
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed classification]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-047
+  Scenario: must get an error response for a malformed roles
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [for a malformed roles]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-048 @Ignore # This is the test need to be done manually
+  Scenario: must get an error response for a upload document with unauthorised user id after 10 minuts
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [contains unauthorised user id]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a positive response is received
+    And   the response has all the details as expected
+    And   the request [contains the id of the document just uploaded above and wait for 10 minuts]
+    And   it is submitted to call the [Get Document Metadata by Document ID] operation of [CCD Case Document AM API]
+    Then  a negative response is received
+    And   the response has all the details as expected
+
+  @S-049
+  Scenario: must successfully Post Upload multiple Document with Binary Content
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
+    When  a request is prepared with appropriate values
+    And   the request [contain multiple document with binary content]
+    And   it is submitted to call the [Post Upload Document with Binary Content] operation of [CCD Case Document AM API]
+    Then  a positive response is received
+    And   the response [contains the metadata for the document uploaded above]
+    And   the response has all other details as expected
+
+  @S-050
+  Scenario: must get an error response for a above max allowed size of a document
+    Given a user with [an active caseworker profile in CCD with full permissions on a document field]
```

Comment:
I think, these scenarios are not updated on JIRA. please do that. considering JIRA 578 already merged, we should be able to run all and finalise the test data for all the scenarios of upload API.

Reply:
added in jira